CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 
 ## Unreleased
 
+## Dangerzone 0.5.1
+
+### Changed
+
+- Use more descriptive button labels in update check prompt ([issue #527](https://github.com/freedomofpress/dangerzone/issues/527), thanks to [@garrettr](https://github.com/garrettr))
+
+### Removed
+
+- Platform support: Drop Fedora 37, since it reached end-of-life ([issue #637](https://github.com/freedomofpress/dangerzone/issues/637))
+
+### Security
+
+- Protect our container image against CVE-2023-43115, by updating GhostScript to version 10.02.0. Note that this CVE affects the **untrusted** environment where the conversion of the document to pixels takes place. Dangerzone operates under the assumption that this environment will eventually get exploited, which is why it protects the users in depth by running this environment in a hardened container, as a defense in depth measure. We are not aware of any container escape that impacts our users' security, but it's highly recommended to update to the latest Dangerzone version.
```

Comment:
We should probably add the advisory date here as well, once we draft it. Most probably it would be today's date.

Reply:
I've pushed a fixup commit that adds the 2023-12-07 advisory, and adds some links.